docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -1,118 +1,123 @@
 ---
-title: Manage NuGet packages using the nuget.exe CLI
-description: Instructions for using the nuget.exe CLI to work with NuGet packages.
+title: Manage NuGet packages with the NuGet CLI
+description: Instructions for using the NuGet CLI, nuget.exe, to install, list, update, remove, and restore NuGet packages.
 author: mikejo5000
 ms.author: mikejo
-ms.date: 06/03/2019
+ms.date: 08/29/2022
 ms.topic: conceptual
 ---
 
-# Manage packages using the nuget.exe CLI
+# Manage NuGet packages with the NuGet CLI
 
-The CLI tool allows you to easily update and restore NuGet packages in projects and solutions. This tool provides all NuGet capabilities on Windows, and also provides most features on Mac and Linux when running under Mono.
+You can use the `nuget.exe` CLI tool to install, list, update, remove, and restore NuGet packages in Visual Studio projects and solutions. The NuGet CLI provides all NuGet capabilities on Windows, and provides most NuGet capabilities on Mac and on Linux with Mono. This article describes the most common NuGet CLI commands for managing NuGet packages.
```

Comment:
NuGet.exe can't do all the listed things. 

There's no proper list and remove functionalities. 
I think we should only say `restore` and the install/update/remove commands are completed through the PM UI.

List just happens to work for packages.config projects because of a design decision, but it's not really guaranteed to be 100% correct every time

Reply:
@nkolev92 - How about "manage," as that's what the article states it's about, and if we only say "restore" people might think they're in the wrong article.

Reply:
Yep, great idea.